.github/scripts/lgtm-processor.js

```
@@ -0,0 +1,327 @@
+ * @param {string[]} wildcardRoles
+ * @returns {{ requiredReviewerRoles: Set<string>, hasFilesWithoutSpecificOwners: boolean }}
+ */
+function getRequiredReviewerRoles(changedFiles, codeownerMappings, wildcardRoles) {
+  const requiredReviewerRoles = new Set();
+  let hasFilesWithoutSpecificOwners = false;
+
+  changedFiles.forEach(file => {
+    let hasSpecificOwner = false;
+    codeownerMappings.forEach(mapping => {
+      if (fileMatchesPattern(file, mapping.pattern)) {
+        mapping.roles.forEach(role => requiredReviewerRoles.add(role));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)